Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -832,7 +834,7 @@ tasks:
       brand: Builtin
     nexttasks:
       '#none#':
-      - "136"
+      - "110"
```

Comment:
change back to the way it was.  Changes in https://github.com/demisto/content/pull/28908

Reply:
resolved